projects/ngx-sub-form/src/lib/ngx-sub-form.ts

```diff
@@ -148,22 +148,19 @@ export function createForm<ControlInterface, FormInterface>(
       if (!isRoot<ControlInterface, FormInterface>(options)) {
         return formGroup.valueChanges.pipe(delay(0));
       } else {
-        if (options.manualSave$) {
-          return options.manualSave$.pipe(
-            withLatestFrom(formGroup.valueChanges),
-            map(([_, formValue]) => formValue),
-            filter(() => formGroup.valid),
-            delay(0),
-            filter(formValue => !isEqual(transformedValue, formValue)),
-          );
-        } else {
-          return formGroup.valueChanges.pipe(
-            filter(() => formGroup.valid),
-            delay(0),
-            filter(formValue => !isEqual(transformedValue, formValue)),
-            options.handleEmissionRate ?? identity,
-          );
-        }
+        const formValues$ = options.manualSave$
+          ? options.manualSave$.pipe(
+              withLatestFrom(formGroup.valueChanges),
+              map(([_, formValue]) => formValue),
+            )
+          : formGroup.valueChanges;
```

Comment:
[minor] could be written as the following to avoid the repetition

```ts
const formValues$ =
  options.manualSave$?.pipe(
    withLatestFrom(formGroup.valueChanges),
    map(([_, formValue]) => formValue)
  ) ?? formGroup.valueChanges;
```

Reply:
ah yea I wondered about this one, I kinda feel a ternary is slightly clearer in terms of readability, but on the fence about that. Happy for it to be done at a later date